packages/manager/apps/web/client/app/hosting/email/activate/index.js

```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+
+import addModule from './activate.module';
+
+const moduleName = 'webHostingEmailActivateComponent';
+
+angular.module(moduleName, [
+  'ui.router',
```

Comment:
No need for translations or ui-kit ?

Reply:
translations are included in module

Reply:
So you should use the angular translate module :)